# Tentative programme

### Monday 9 December

| 9:00 - 9:15   | Registration                                                                                                                       |
| 9:15 - 9:30   | Welcome and introductions                                                                                                          |
| 9:30 - 10:00  | Invited talk 1: Martin Morgan - *The Bioconductor Project: Updates and Directions*                                                 |
| 10:00 - 10:30 | Invited talk 2: Andrea Rau - *Individualized multi-omic pathway deviation scores using multiple factor analysis*                   |
| 10:30 - 11:00 | Break + [poster session](#posters)                                                                                                 |
| 11:00 - 12:00 | [Flashlight session 1](#session-1) (7 x 8 minutes)                                                                                 |
| 12:00 - 12:30 | Invited talk 3: Vince Carey - *Software ecosystem concepts for federated genomic analysis*                                         |
| 12:30 - 13:30 | Lunch  + [poster session](#posters)                                                                                                |
| 13:30 - 14:30 | [Flashlight session 2](#session-2) (7 x 8 minutes)                                                                                 |
| 14:30 - 15:00 | Invited talk 4: Levi Waldron - *Multi'omic analysis of subtype evolution and heterogeneity in high-grade serous ovarian carcinoma* |
| 15:00 - 15:30 | Invited talk 5: Lieven Clement - *The pitfalls of summarisation and imputation on label free mass spectrometry based proteomics*   |
| 15:30 - 16:00 | Coffee break + [poster session](#posters)                                                                                          |
| 16:00 - 16:30 | Invited talk 6: Christine Choirat - *Enabling Practical Reproducibility With the RENKU Platform*                                   |
| 16:30 - 17:02 | [Flashlight session 3](#session-3) (4 x 8 minutes)                                                                                 |
| 17:02 - 17:15 | Presentation of group activities and wrap-up.                                                                                      |

**Evening**:

| 18:30 - 21:30 | Dinner at the [Hof ter Musschen](https://www.hoftermusschen.be/en/) (located at walking distance from the conference venue) |


### Tuesday 10 December

| 9:00 - 10:00  | [Flashlight session 4](#session-4) (7 x 8 minutes)                                                                |
| 10:00 - 10:30 | Invited talk 7: Ewa Szczurek - *Linear effects models of signaling pathways from combinatorial perturbation data* |
| 10:30 - 11:00 | Break + [poster session](#posters)                                                                                |
| 11:00 - 12:30 | Group activities                                                                                                  |
| 12:30 - 13:30 | Lunch + [poster session](#posters)                                                                                |
| 13:30 - 14:00 | Group activities presentations                                                                                    |
| 14:00 - 14:30 | Invited talk 8: Davide Risso - *Efficient and scalable analysis of single-cell RNA-seq data using Bioconductor*   |
| 14:30 - 15:00 | Invited talk 9: Elisabetta Mereu - *Benchmarking Single-Cell RNA Sequencing Protocols for Cell Atlas Projects*    |
| 15:00 - 15:30 | Invited talk 10: Andreas Moor - *Spatially resolved transcriptomics in mammalian tissues*                         |
| 15:30 - 16:00 | Wrap-up and closing of conference                                                                                 |
| 16:00 - 17:00 | Preparation of user/developer sessions                                                                            |

### Wednesday 11 December (optional)

| 9:00 - 10:30  | [User/developer sessions](https://github.com/bioconductor/eurobioc2019/issues) |
| 10:30 - 11:00 | Break                                                                          |
| 11:00 - 12:30 | User/developer sessions (cont.)                                                |
| 12:30 - 13:30 | Lunch                                                                          |
| 13:30 - 15:00 | User/developer sessions (cont.)                                                |
| 15:00 - 16:00 | Sessions feedback and wrap-up                                                  |

## Flashlight talks
### Session 1

* Witold Wolski - *fgczgseaora: unifying methods on gene set enrichment*
* Federico Marini - *GeneTonic: combining components of transcriptome analyses for efficient exploration*
* Ozan Cinar - *poolr: An extensive set of methods for gene-based testing*
* Nathalie Pochet - *AMARETTO for network biomedicine: linking diseases, drivers and drugs via multi-omics, clinical, imaging and perturbation data fusion*
* Theo Killian - *Exploiting the Depmap cancer dependency data using the depmap R package*
* Mike Smith - *Benchmarking compression filters for HDF5 backed matrices*
* Anne Bras - *flowCore - lossless compression*

### Session 2

* Katarzyna Gorczak - *Transcriptomics data visualization*
* Ruizhu Huang - *Visualizing TreeSummarizedExperiment objects and tree climbing with treeclimbR*
* Michela Pasetto - *transcriptomicsTemplates: a template for the analysis of transcriptomics data*
* Simone Tiberi - *BANDITS: Bayesian differential splicing accounting for sample-to-sample variability and mapping uncertainty*
* Sara Fonseca Costa - *Expression calls from RNA-seq using BgeeCall R package*
* Katharina Hembach - *exondiscovery: prediction of unannotated exons from splice-junction RNA-seq reads*
* Robert Castelo - *GenomicScores: latest developments and future challenges*

### Session 3

* Johannes Rainer - *Flexible infrastructure for mass spectrometry data*
* Christian Panse - *Direct access to mass spectrometry data in R*
* Oliver Crook - *Uncertainty quantification for dynamic spatial proteomics experiments*
* Nils Kurzawa - *Robust statistical inference of small molecule-protein interactions from 2D thermal proteome profiles reveals off-targets of epigenetic drugs*

### Session 4

* Dario Righelli - *Epigenomic enrichment analysis using Bioconductor*
* Dania Machlab - *monaLisa: MOtif aNAlysis with Lisa*
* Stephany Orjuela - *DAMEfinder: A method to screen for changes in allele-specific methylation*
* Axelle Loriot - *Genome-wide approach to identify new transcripts induced by DNA hypomethylation in tumors*
* Almut Luetge - *Explore batch effects and data integration with CellMixS*
* Jeroen Gilis - *Scalable differential transcript usage analysis for single-cell applications*
* Pierre-Luc Germain - *A framework for pipeline benchmarking and its application to scRNAseq clustering*

## Posters

- Chong Tang: *msdata2: Lable-Free Proteomics data for Benchmarking*
- Christophe Vanderaa: *scpdata: a data package for single-cell proteomics*
- Gibb Sebastian: *Spectra: a flexible infrastructure for mass spectrometry data*
- Teun van den Brand: *GENOVA: Explore the Hi-Cs*
- Tore Bleckwehl: *Single cell methods for regulatory elements*
- Mireia Ramos-Rodríguez: *UMI4Cats: an R package for analyzing UMI-4C
  contact data*
- Liesbeth François: *Insights into biological mechanisms of an in
  vitro epilepsy model by leveraging gene coexpression modules*
- Aliaksandra Kakoichankava: *Deconvolution of transcriptomic data
  shows biologically and clinically relevant signals in pancreatic
  tumors*
- Tobias Kockmann & Christian Panse: *Direct access to mass spectrometry data in R - the rawR package*
- Laurent Gatto: *The R for Mass Spectrometry Initiative*
- Katarzyna Górczak: *Transcriptomics data visualization*
- Pierre Luc Germain: *A framework for pipeline benchmarking and its
  application to scRNAseq clustering*
- Michela Pasetto: *transcriptomicsTemplates: a templates for the
  analysis of transcriptomics data*
- Sara Fonseca Costa: *Expression calls from RNA-Seq data using
  BgeeCall R package*
- Nathalie Pochet: *AMARETTO for network biomedicine: linking
  diseases, drivers and drugs via multi-omics, clinical, imaging and
  perturbation data fusion*
- Theo Killian: *Exploiting the Depmap cancer dependency data using
  the depmap R package*
- Mike Smith: *Benchmarking compression filters for HDF5 backed
  matrices*
- Petr Nazarov: *consICA: a package for parallel consensus ICA of
  multi-omics data and interpretation of the results*